README.md

```diff
@@ -31,23 +31,20 @@ There are several optional dependencies that enable quality-of-life features.
 
 ## Resque
 
-TBD
+Resque support is currently in the works.
 
 ## Sidekiq
 
 TransferJobs supports common Sidekiq job configurations. The provided classes and modules should enable the construction of more complex or atypical flows. The provided `sidekiq:transfer` rake task supports the following out-of-the-box:
 
 - sidekiq ~> 5.0
-- sidekiq-unique-jobs ~> 5.0.5
-- sidekiq-scheduler ?
+- sidekiq-unique-jobs ~> 5.0.10
 
 ### Caveats
 
+If you are a user of Sidekiq Pro or Sidekiq Enterprise, please be aware of these restrictions. Currently, we don't support all the functionality provided by Pro and Enterprise, notably, expiring jobs. You will need to audit your app for these features before using TransferJobs
```

Comment:
Is expiring jobs the only caveat or are there others? 
This is worded to vaguely sound like there are other caveats but we have not listed them here. Is this on purpose to cover unknowns?

Reply:
We could probably just mention what we explicitly support and everything is should be considered accidentally supported (if at all).

Reply:
Yea that seems smarter. I tried to read through the ent and pro source to figure out what could break but really I can't be certain.